File: Pattern/4.py
# 1
# 2 2
# 3 3 3
# 4 4 4 4
# 5 5 5 5 5


n = int(input("Enter number: "))

for i in range(n):
    for j in range(i + 1):
        # just print the row number
        print(i + 1, end=" ")  # i will start from 0 thats why we are printing i+1
    print()

File: Patterns in JS/4.js
// 1
// 1 2
// 1 2 3
// 1 2 3 4
// 1 2 3 4 5

function four(n) {
  for (let i = 0; i < n; i++) {
    let row = "";
    for (let j = 0; j < i + 1; j++) {
      row += j + 1 + " ";
    }
    console.log(row);
  }
}
four(5);
